src/utils/transformCommonComponents.ts

```diff
@@ -0,0 +1,84 @@
+import { CommonComponents } from '@prisma/client';
+
+function tranformCommonComponentOption(original) {
+  if (original === undefined || original === null) {
+    return {
+      planningToUse: false,
+      implemented: false,
+    };
+  } else if (original === 'PLANNING_TO_USE') {
+    return {
+      planningToUse: true,
+      implemented: false,
+    };
+  } else if (original === 'IMPLEMENTED') {
+    return {
+      planningToUse: false,
+      implemented: true,
+    };
+  } else {
+    return {
+      planningToUse: false,
+      implemented: false,
+    };
+  }
+}
+
+export const transformCommonComponents = (originalComponents: any) => {
```

Comment:
I see the same helper function `transformCommonComponents` in one of the migration script.
Is there any chance that we can use this in the migration script to reduce duplication?

Reply:
If it takes some efforts as the script will be used only one time, maybe okay to leave it as it is? 🤔

Reply:
I tried again and it works!